isis/scripts/downloadIsisData

```diff
@@ -159,35 +157,51 @@ def main(mission, dest, cfg_path, dry_run, ntransfers):
     else:
         for remote in supported_missions[mission]:
-            args = create_rclone_arguments(dest, remote, dry_run, ntransfers)
-            rclone(command="sync", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
+            args = create_rclone_arguments(dest, remote,  ntransfers, kwargs)
+            rclone(command="copy", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
 
 
 if __name__ == '__main__':
+    usageString = "usage: downloadIsisData mission dest [-h] [-v] [-n NUM_TRANSFERS] [--config CONFIG] [rclone flags]"
     helpString = (
-    '''This will allow for a user to download isis data directly to their machine from the USGS S3 buckets as well as public end points\n\n'''
-    '''<Mission name> <destination to copy to> <--dry-run (optional)> \n\n'''
-    '''Example of how to run this program:\n\n'''
-    '''    downloadIsisData tgo ~/isisData/\n\n'''
-    '''NOTE: if you would like to download the data for every mission supply the value ALL for the <Mission name> parameter''')
-    parser = argparse.ArgumentParser(description = helpString, formatter_class=argparse.RawTextHelpFormatter)
-    parser.add_argument('mission', help='mission for files to be downloaded')
+    '''
+    Use to copy ISIS data from USGS sources as well as public end points. 
+
+    This script combines multiple sources using the rclone command under the hood. 
+    
+    After positional args, accepts arbitrary rclone flags. Run "rclone help flags" to get a full list of rclone flags. 
+    
+    See the ISIS repo (https://github.com/USGS-Astrogeology/ISIS3) for more info. 
+    
+    Examples of how to run this program:
+        
+        # download everything for tgo
+        $ downloadIsisData tgo $ISISDATA
+    
+        # download everythin except kernels with debug output
+        $ downloadIsisData all $ISISDATA --dry-run --exclude="kernels/**" -vv
+        
+        # download only cks and fks
+        $ downloadIsisData all $ISISDATA --include="{ck/**,fk/**}"''')
+    parser = argparse.ArgumentParser(description = helpString,formatter_class=argparse.RawDescriptionHelpFormatter, usage=usageString)
+    parser.add_argument('mission', help='mission for files to be downloaded. Use "all" to download everything, "legacybase" to download base data for ISIS<4.1.0.')
     parser.add_argument('dest', help='the destination to download files from source')
-    parser.add_argument('--dry-run', help="run a dry run for rclone value should be a boolean", default=False, action='store_true')
     parser.add_argument('-v', '--verbose', action='count', default=0)
     parser.add_argument('-n', '--num-transfers', action='store', default=10)
     parser.add_argument('--config', action='store', default=find_conf())
-    args = parser.parse_args()
+    args, kwargs = parser.parse_known_args()
+    
+    print(kwargs)
```

Comment:
Do you want this print in here? Maybe a logger with debug is better?

Reply:
This was temporary so I easily look at them without them being instantly swallowed in the pile of debug prints when -vv is passed. Should have been removed.